crates/colorchoice-clap/src/lib.rs

```diff
@@ -11,6 +11,8 @@
 //! }
 //! ```
 
+use clap::ColorChoice;
```

Comment:
Speaking of, this is a breaking change that isn't called out anywhere.

Reply:
Removing the `pub` was a mistake. I don't mind adding it back. But I was wondering, if we need it at all. What were the original reasons in adding it?

Reply:
Breaking changes should have at least been called out.

Though for now, I'm wanting to hold off on breaking changes.  If you still want to do a breaking change, I'd recommend we sit on this a while.